### PR TITLE
Formatting updates for Advanced Config Docs

### PR DESCRIPTION
**Pull request by ned1313**
Trying to bring formatting consistency with user of headers, horizontal rules, notices, and versioned link shortcode.

### CODE REVIEW

File: docs/content/advanced_configuration/consul_kv.md [hunk 1]
```diff
@@ -4,32 +4,25 @@ weight: 50
...
 When desiring to run without Kubernetes, it is possible to instead provide this file to Gloo inside of a configuration directory.
 
-When running the `gloo`, `discovery`, and `gateway` processes, it is necessary to provide a `--dir` flag pointing to
-the config directory containing the Settings YAML.
+When running the `gloo`, `discovery`, and `gateway` processes, it is necessary to provide a `--dir` flag pointing to the config directory containing the Settings YAML.
 
 If we were to create a directory called `data`, the structure of the directory should look like so:
 
 ```bash
- tree data
```
Comment (kdorosh) [inline docs/content/advanced_configuration/consul_kv.md]: `tree` is the command to run to generate the output below, we should keep no

Comment (ned1313) [reply docs/content/advanced_configuration/consul_kv.md]: I don't think we need to show the command, the directory structure is what's important. We don't show the `tree` command in other docs.

File: docs/content/advanced_configuration/session_affinity.md [hunk 2]
```diff
@@ -22,20 +18,19 @@ There are two steps to configuring session affinity:
   - This can include any combination of headers, cookies, and source IP address.
 
 
-Below, we show how to configure Gloo to use hashing load balancers and demonstrate a common cookie-based
-hashing strategy using a Ring Hash load balancer.
-
+Below, we show how to configure Gloo to use hashing load balancers and demonstrate a common cookie-basedhashing strategy using a Ring Hash load balancer.
```
Comment (kdorosh) [inline docs/content/advanced_configuration/session_affinity.md]: typo, 
```suggestion
Below, we show how to configure Gloo to use hashing load balancers and demonstrate a common cookie-based hashing strategy using a Ring Hash load balancer.
```

File: docs/content/advanced_configuration/session_affinity.md [hunk 3]
```diff
@@ -147,40 +139,35 @@ spec:
...
-The counter simply reports how many requests have been made to the `/count` endpoint.
-Without session affinity, subsequent requests will return a non-monotonically increasing response.
-For example, on a fresh deployment, your first request will be handled by node 1, and return a count of 1.
-Your second request will by handled by node 2, and also return a count of 1.
-After you enable session affinity, repeat requests will return a strictly increasing count response.
+We will use a very simple "counter" app to demonstrate session affinity configuration. The counter simply reports how many requests have been made to the `/count` endpoint. Without session affinity, subsequent requests will return a non-monotonically increasing response. For example, on a fresh deployment, your first request will be handled by node 1, and return a count of 1. Your second request will by handled by node 2, and also return a count of 1. After you enable session affinity, repeat requests will return a strictly increasing count response.
+
+The source code for the session affinity app is available [here](https://github.com/solo-io/gloo/blob/v1.2.12/docs/examples/session-affinity/main.go).1
```
Comment (kdorosh) [inline docs/content/advanced_configuration/session_affinity.md]: typo,
```suggestion
The source code for the session affinity app is available [here](https://github.com/solo-io/gloo/blob/v1.2.12/docs/examples/session-affinity/main.go).
```

**Pull request merged**